pkg/scheduler/api/resource_info.go

```diff
@@ -429,6 +429,42 @@ func (r *Resource) LessEqual(rr *Resource, defaultValue DimensionDefaultValue) b
 	return true
 }
 
+// LessEqualWithDimension only compare the resource items in req param
```

Comment:
I think we should clarify that this func behaves when one dimension of request missed in resource1 and when no dimension missed in resource1 but less than resource1.

Reply:
seems that we only consider whether the dimension exists but not care the specific value of it, right?

Reply:
> seems that we only consider whether the dimension exists but not care the specific value of it, right?

yes